guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -5,7 +5,7 @@ include::common/header.adoc[]
 
 = {UpgradingDocTitle}
 
-ifeval::["{build}" == "satellite" && "{backend}" == "html5"]
+ifdef::satellite[]
 include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
 endif::[]
```

Comment:
Can we move this lower with the other includes below the term definitions?

Reply:
Done :+1: